### ISSUE TITLE
SVN `/trunk` is not used when creating a new site via CLI wizard

### ISSUE THREAD

**Issue by ntwb** [bradp/vv#285]
How I created the new site:
```shell
$  vv create --path /Users/netweb/dev/vvv
New VVV Site Setup 
 Name of new site directory: multisite-bbpress
 Domain to use (leave blank for multisite-bbpress.dev): 
 WordPress version to install (leave blank for latest version or trunk for trunk/nightly version): trunk
 Install as multisite? (y/N): y
 Install as subdomain or subdirectory? : subdomain
 Git repo to clone as wp-content (leave blank to skip): 
 Local SQL file to import for database (leave blank to skip): 
 Remove default themes and plugins? (y/N): 
 Add sample content to site (y/N): 
 Enable WP_DEBUG and WP_DEBUG_LOG (y/N): y

About to perform the following: 
* Halt Vagrant (if running)
* Create directory multisite-bbpress in /Users/netweb/dev/vvv/www
* Create files vvv-init.sh, wp-cli.yml, and vvv-hosts in directory multisite-bbpress
* Create file multisite-bbpress.conf in /Users/netweb/dev/vvv/config/nginx-config/sites
* Run `vagrant up --provision` to initialize site

Provisioning Vagrant will do the following: 
* Create database multisite-bbpress
* Install WordPress (trunk) in the htdocs directory
* Make the site visible at multisite-bbpress.dev

 Continue (y/n)? : y
[Success] New VVV setup starting for site 'multisite-bbpress' 
Checking to see if vagrant halt should be run... 
Vagrant not running, skipping `vagrant halt`... 
Creating site directory, wp-cli.yml, and vvv-init.sh file...  
Done
Adding multisite-bbpress.dev to new vvv-hosts file...  
Done
Creating nginx-config/sites/multisite-bbpress.conf...  
Done
Running vagrant up --provision...  
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Checking if box 'ubuntu/trusty64' is up to date...
==> default: A newer version of the box 'ubuntu/trusty64' is available! You currently
==> default: have version '20160406.0.0'. The latest is version '20160602.0.0'. Run
==> default: `vagrant box update` to update.
==> default: VirtualBox VM is already running.
==> default: Checking for host entries
==> default: adding to (/etc/hosts) : 192.168.50.4  multisite-bbpress.dev  # VAGRANT: 8434e946173a47257d90b7c82fd64da2 (default) / dfe74b72-edc3-446e-8ea7-d71d8fb60d6d
Password:
==> default: Running triggers after up...
==> default: Executing command "vagrant ssh -c vagrant_up"...
==> default: Command execution finished.
[Success] New VVV Site Setup: Done! 
Directory: /Users/netweb/dev/vvv/www/multisite-bbpress
URL:       multisite-bbpress.dev
Username:  admin
Password:  password

```

Excerpt of the provisioning process:
```shell
==> default: Installing WordPress (trunk) in multisite-bbpress/htdocs...
==> default: Downloading WordPress 4.5.2 (en_US)...
==> default: Using cached file '/root/.wp-cli/cache/core/wordpress-4.5.2-en_US.tar.gz'...
==> default: Success: WordPress downloaded.
==> default: Success: Generated wp-config.php file.
==> default: Created single site database tables.
==> default: Set up multisite database tables.
==> default: Added multisite constants to wp-config.php.
==> default: Success: Network installed. Dont forget to set up rewrite rules.
==> default: /srv/www/multisite-bbpress
==> default: A    trunk/wp-load.php
==> default: A    trunk/wp-includes
==> default: A    trunk/wp-includes/formatting.php
==> default: A    trunk/wp-includes/class.wp-styles.php
==> default: A    trunk/wp-includes/class-wp-metadata-lazyloader.php
...
svn checkout continues...
...
==> default: A    trunk/wp-content/plugins/akismet/LICENSE.txt
==> default:  U   trunk/wp-content/plugins/akismet
==> default: Checked out external at revision 1434235.
==> default: Checked out revision 37639.
```

Once the SVN checkout above is complete this is created in: 
• `/Users/netweb/dev/vvv/www/multisite-bbpress/trunk`

Though Nginx is set to the `htdocs` folder as the webroot:
• `/Users/netweb/dev/vvv/www/multisite-bbpress/htdocs`

My `vvv-init.sh` created is:

```bash

if [ ! -d "htdocs/wp-admin" ]; then
	echo 'Installing WordPress (trunk) in multisite-bbpress/htdocs...'
	if [ ! -d "./htdocs" ]; then
		mkdir ./htdocs
	fi
	cd ./htdocs
	wp core download --locale=en_US --allow-root 
	wp core config --dbname="multisite-bbpress" --dbuser=wp --dbpass=wp --dbhost="localhost" --dbprefix=wp_ --locale=en_US --allow-root --extra-php <<PHP
define('WP_DEBUG', true);
define('WP_DEBUG_DISPLAY', false);
define('WP_DEBUG_LOG', true);
define('SCRIPT_DEBUG', true);
define('JETPACK_DEV_DEBUG', true);
PHP
	wp core multisite-install --subdomains --url=multisite-bbpress.dev --title="multisite-bbpress" --admin_user=admin --admin_password=password --admin_email=admin@localhost.dev --allow-root
	
	
  
  
	
	
	cd -

fi
svn checkout https://core.svn.wordpress.org/trunk/
```

So wp-cli is downloading `core` into `/htdocs`, then an SVN checkout to `/trunk` but the SVN checkout is never copied from `/trunk` to `/htdocs`

I confirmed the SVN revision [r37673](https://core.trac.wordpress.org/changeset/37673) change is in the SVN checkout, but it is not in `/htdocs`

_Sorry about the big walls of shell code..._

**Comment by ntwb** [comment]
As a workaround, I've manually copied WP from `/trunk` to `/htdocs` and I've now got:
• `You are using a development version (4.6-alpha-37673). Cool! Please stay updated.`

Rather than:
• `WordPress 4.5.2`

**Comment by afragen** [comment]
Same issue here, https://github.com/bradp/vv/issues/265

**Comment by ntwb** [comment]
Oh, my bad, I even looked at #265 😞 , thanks @afragen 

I'll leave this open for now, @bradp can decide if to keep it open or not, I'm happy to see everything done in #265 and close this as a dupe 😄

**Comment by afragen** [comment]
This really is fixed, but not updated to Homebrew. Refer to the fix in #265 and manually apply until Homebrew is updated.

**Comment by ntwb** [comment]
Closing...
